### ISSUE TITLE
Archived conversations still show up in share dialog

### ISSUE THREAD

**Issue by apfelbox** [wireapp/wire-ios#1680]
## Checklist (please check before submitting)

* [x] I checked the existing issues, and the issue was not reported before.

## The problem

I have three archived conversations in wire.
If I want to share something and am selecting the conversation, these archived conversations still show up in the selector.

## Environment

* Wire version that exhibits the issue: 3.6 (2962)
* Last Wire version that did not exhibit the issue (if applicable): not sure
* iOS version used to run Wire: 11.1.2 (15B202)
* iOS device type: iPhone 7+
* Mobile network type (EDGE/LTE/Wi-Fi/Offline): not relevant
* Exact date, time and timezone (only for the crash issues): not relevant

## Details

1. Archive a conversation
2. go to any app
3. Share something
4. Select wire
5. Open the conversation selector
6. See your archived conversation

A plus would be, that if one only has a single active conversation, it would be auto-selected in the share dialog.

## Calling Debug logs (only for calling issues)

Not relevant

## Screenshots

(I can provide them, if they are needed, but I think that this issue should be clear enough as it is)

**Comment by mikeger** [comment]
Hey @apfelbox ! Thanks for the report, as I understand the problem, you would prefer not to see the archived conversations in the list in the share extension?

**Comment by apfelbox** [comment]
Yes, exactly. If I would decide, that I want to share something with them again, I would have to unarchive the conversation.

But that is okay. Currently the archived discussions just clutter up the selector.
